### ISSUE TITLE
suggestion 1.20.1 

### ISSUE THREAD

**Issue by Vampzula** [Traben-0/stuck_arrows_for_all#1]
Hi! Very cool idea, but I would like to know if it is possible to make a version for Fabric 1.20.1 too?

**Comment by TheGamerCailo** [comment]
Yeah, I would love a Fabric 1.20.1 version too, I'm trying to add this to my modpack. Therefore, I am bumping lol.

**Comment by Beez3-A** [comment]
Awesome mod!!

Same 1.20.1 would be real nice

**Comment by Traben-0** [comment]
I am considering it, though 1.21.3 will come first

**Comment by Wittummm** [comment]
I would really like this on 1.20.1 too! Bump!

**Comment by Sylthena** [comment]
1.20.1 would be amazing!

**Comment by Beez3-A** [comment]
Bump Again, This mod is too good to not be on 1.20.1. It's the mod that completes my mod list. Please @Traben-0

**Comment by MinecraftGuy926** [comment]
bump :p

**Comment by Promikbbb** [comment]
I also needed version 1.20.1, so I decided to create my own fork. You can check out my fork: https://github.com/Promikbbb/Stuck_Arrows_For_All_1.20.1

**Comment by MinecraftGuy926** [comment]
You're a legend!
